.env

```
@@ -0,0 +1,6 @@
+# Supabase → Settings → API から取得
+PUBLIC_SUPABASE_URL=https://jxrfsbhluooyrkrooger.supabase.co
+PUBLIC_SUPABASE_ANON_KEY=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZSIsInJlZiI6Imp4cmZzYmhsdW9veXJrcm9vZ2VyIiwicm9sZSI6ImFub24iLCJpYXQiOjE3NzYwODc3MzEsImV4cCI6MjA5MTY2MzczMX0.9RKZNSFNzKgzHspq3zA0CahJCd4_zsUgvPVAcTRGwGA
+SUPABASE_SERVICE_ROLE_KEY=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZSIsInJlZiI6Imp4cmZzYmhsdW9veXJrcm9vZ2VyIiwicm9sZSI6InNlcnZpY2Vfcm9sZSIsImlhdCI6MTc3NjA4NzczMSwiZXhwIjoyMDkxNjYzNzMxfQ.eoj-Srm3ktgvxIgeo3yAYmAZEYIQ1D9wehzNiAdLMdM
+# 管理者メールアドレス（アニメカバー画像アップロード権限）
+ADMIN_EMAIL=iwazakihikaru@gmail.com
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The ADMIN_EMAIL key should go before the PUBLIC_SUPABASE_ANON_KEY key

(UnorderedKey)